sentry_sdk/client.py

```diff
@@ -74,23 +74,37 @@ class _Client(object):
     """
 
     def __init__(self, *args, **kwargs):
-        # type: (*Optional[str], **Any) -> None
+        # type: (*Any, **Any) -> None
+        self.options = get_options(*args, **kwargs)  # type: Dict[str, Any]
+        self._init_impl()
+
+    def __getstate__(self):
+        # type: () -> Any
+        return self.options
```

Comment:
can we return `{'options': self.options}` here instead? I don't like the idea of reusing the state dictionary directly for the options as otherwise we can't add more stuff later that are not options if needed.